### ISSUE TITLE
Cannot get image manifest or delete tag from a private registry

### ISSUE THREAD

**Issue by AxelVoitier** [microsoft/vscode-docker#4348]
Hello,

Using extension version 1.29.1.

I have a private registry setup (pretty much barebone from distribution, version 2.8.3, just having TLS enabled). I cannot get the manifest of any image in that registry. Neither can I delete any tag (because apparently the first step in deleting is fetching the manifest).

Error given in vscode is "failed with status 404: Not Found".
Looking at the registry logs, I get the following:
```
registry-1  | 10.x.x.x - - [26/Jul/2024:09:50:52 +0000] "GET /v2/USER/IMAGE/manifests/TAG HTTP/1.1" 404 117 "" "node"
registry-1  | time="2024-07-26T09:50:52.803491926Z" level=error msg="response completed with error" err.code="manifest unknown" err.message="OCI index found, but accept header does not support OCI indexes" go.version=go1.20.8 http.request.host="ADDRESS:PORT" http.request.id=a6497708-9e95-49fe-ab1e-345a17fd04f4 http.request.method=GET http.request.remoteaddr="10.x.x.x:59214" http.request.uri="/v2/USER/IMAGE/manifests/TAG" http.request.useragent=node http.response.contenttype="application/json; charset=utf-8" http.response.duration=3.656995ms http.response.status=404 http.response.written=117 vars.name="USER/IMAGE" vars.reference=TAG 
``` 

If I search on this OCI accept header error, I get the following:
https://my.f5.com/manage/s/article/K000138411
Basically saying the request should be made with two particular accept headers:
`-H "Accept: application/vnd.oci.image.manifest.v1+json" -H "Accept: application/vnd.oci.image.index.v1+json"`

Possibly related to https://github.com/microsoft/vscode-docker/issues/4309#issuecomment-2138044818

Cheers,
Axel

**Comment by AxelVoitier** [comment]
Testing direct access to the registry with curl, I indeed need to accept `application/vnd.oci.image.index.v1+json` content to retrieve it.
See https://github.com/opencontainers/image-spec/blob/main/image-index.md.

This seems to be because I am storing multiplatform images in that registry. Or, actually, in some cases the images are single platform (ARM), but they were built using a multiplatform-enabled buildx builder.

**Comment by danegsta** [comment]
This should be resolved with the updated `@microsoft/vscode-docker-registries` package that was just merged

**Comment by danegsta** [comment]
Release 1.29.2 was just published to the marketplace with a fix for this issue.

**Comment by AxelVoitier** [comment]
Hello,

I can confirm fetching manifest and deleting multiplatform image works now. Thanks :+1: .


Cheers,
Axel
